strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-composer-layout-provider/src/test/java/org/carlspring/strongbox/artifact/coordinates/ComposerArtifactCoordinatesTest.java

```diff
@@ -0,0 +1,96 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import edu.emory.mathcs.backport.java.util.Arrays;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.data.jdbc.DataJdbcTest;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Tests for ComposerArtifactCoordinates class
+ *
+ * @author jcoelho
+ */
+public class ComposerArtifactCoordinatesTest
+{
+
+    private static List<String> packages;
+
+    static
+    {
+        packages = new ArrayList<>();
+    }
+
+    @BeforeAll
+    static void setup()
+    {
+
+
+        packages.add("symfony/polyfill-mbstring");
+
+        packages.add("psr/log");
+
+        packages.add("symfony/polyfill-ctype");
+
+        packages.add("guzzlehttp/psr7");
```

Comment:
Would you mind adding some more test data (or even more tests) checking cases with, or without versions, types and so on? Thanks! :)

Reply:
Hey @carlspring,
 How would you test with version and types?
In Composer, the path does not include version and type. There is not a representation like with Maven `symfony:polyfill:1.2.3`.

Thanks.
;)

Reply:
I have used the syntax `vendor/name/version/type`